examples/simple-platform/run-example.md

```diff
@@ -0,0 +1,144 @@
+# Running the Platform Example Locally
+
+## Setup
+
+### Create a GitHub development application for the `account-app` OAuth process
+
+At this time, the accounts app supports account creation and authentication 
+via GitHub only. In order to run a user authentication loop, you need to 
+create a new GitHub OAuth app and configure it to connect to the 
+`web-auth-endpoint` node.
+
+1. Log into [github.com](https://github.com) and navigate to Settings > 
+Developer settings. 
+1. Under OAuth Apps, create a new OAuth App and give it any application name, 
+e.g. Interbit Test App.
+1. The Authorization callback URL is `http://localhost:8888/oauth/github`.
+1. Click the Register Application button. Leave this browser tab open as we 
+will copy values from here in the steps below.
+
+### Generate a set of private and public keys for `platform-deploy` and `web-auth-endpoint`
+
+1. Open a terminal and install `interbit-cli` with
+```
+npm i -g interbit-cli
+```
+1. Generate an Interbit key pair for `platform-deploy` and `web-auth-endpoint`, 
+respectively, with
+```
+interbit keys --filename platform-deploy-keys.json
+interbit keys --filename web-auth-endpoint-keys.json
+```
+This will create two new files at the root of the Interbit repository with 
```

Comment:
Try to avoid future tense wherever possible, because it implies a promise to the reader that we might not be able to fulfill. In this case, it also creates uncertainty: when should I expect the new files to be created?

The pattern to follow is "do this then that happens".

So: `This creates two new files...`